lib/elixir/lib/access.ex

```diff
@@ -355,10 +358,11 @@ defmodule Access do
   ## Accessors
 
   @doc """
-  Accesses the given key in a map/struct.
+  Returns a function that accesses the given key in a map/struct the way
```

Comment:
It is important to keep the first line short because we use it as summary in the documentation. So what if we break this documentation and the ones below in two parts:

```
Returns a function that accesses the given key in a map/struct.

The returned function is typically passed as an accessor to
`Kernel.get_in/2`, `Kernel.get_and_update_in/3`, and friends.
```

?